lib/mcb/commands/apiv1/find.rb

```diff
@@ -0,0 +1,25 @@
+name 'find'
+summary 'Find a particular provider entry'
+usage 'find [options] <code>'
+param :code
+
+run do |opts, args, _cmd|
+  # We only need httparty for API V1 calls
+  require 'httparty'
+
+  puts "looking for provider #{args[:code]}"
+
+  provider = each_v1_provider(opts).detect do |p|
+    p['institution_code'] == args[:code]
+  end
+
+  if provider.nil?
```

Comment:
Can this be a guard clause?

Reply:
Well, it's a block so we could do:

```ruby
  if provider.nil?
    error "Provider with code '#{args[:code]}' not found"
    next
  end
```

`next` doesn't feel very idiomatic, but it'd get the job done.